tracks/rust/exercises/clock/mentoring.md

```diff
@@ -40,3 +47,13 @@ impl fmt::Display for Clock {
     }
 }
 ```
+
+### Notes
```

Comment:
Next, can you update the "Notes" section to be the "Common Suggestions" section?

For example:

### Common Suggestions

If they implement Debug or PartialEq manually:
```
We can use the `#[derive]` macro for the `Clock` struct instead of implementing features like [Debug](https://doc.rust-lang.org/std/fmt/trait.Debug.html) and [PartialEq](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html#derivable) manually.
```

If they visibly lament the behavior of % on negative arguments:
```
We can use unstable features like [.rem_euclid()](https://doc.rust-lang.org/std/primitive.i32.html#method.rem_euclid).
```

If they already have such functions in their code:
```
We can take it a step above and add a test for it.
```
(this is a good opportunity to show that tests can be in src/lib.rs where they can see private functions, rather than always having to be in tests/)

It is fine if the clock stores both hours and minutes, but if their logic is hopelessly complicated:
```
It might be easier to only store minutes :)

Reply:
I think the suggestions look a bit strange in code fences, especially without line breaks.  Could I suggest:

#### Blockquote with escapes

If they implement Debug or PartialEq manually:

> We can use the \`#[derive]\` macro for the \`Clock\` struct instead of implementing features like \[Debug](https://doc.rust-lang.org/std/fmt/trait.Debug.html) and \[PartialEq](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html#derivable) manually.

or maybe 

#### Code with manual linebreaks and markdown highlighting

If they implement Debug or PartialEq manually:

```markdown
We can use the `#[derive]` macro for the `Clock` struct instead of implementing
features like [Debug](https://doc.rust-lang.org/std/fmt/trait.Debug.html) and
[PartialEq](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html#derivable) manually.
```

---

(or maybe this can be revisited in a later PR that affects numerous exercises)

Reply:
I think the format is so it's easy to just copy and paste from inside the code block to respond to students, but whatever works :)

Reply:
In that case I'll add line breaks :D